mozjs-sys/etc/patches/0036-Redirect_errors_to_hilog_on_OpenHarmony.patch

```diff
@@ -47,7 +47,7 @@ index 3cfc92533..9c487ac45 100644
 +#ifdef ANDROID
 +  __android_log_print(ANDROID_LOG_ERROR, "Gecko", "mozalloc_abort: %s", msg);
 +#elif defined(OHOS)
-+    (void) OH_LOG_Print(LOG_APP, LOG_ERROR, 0, "Gecko",
++    (void) OH_LOG_Print(LOG_APP, LOG_ERROR, 0xEOC4, "Gecko",
```

Comment:
Would it make sense to define a constant for this domain value, maybe in the [init.configure](https://github.com/servo/mozjs/blob/728acdf3d4ce0604e9f75dd1d539dc6f291ccec7/mozjs-sys/mozjs/build/moz.configure/init.configure#L958), rather than hard coding it?

Reply:
Yeah, that is definitely better than copying the same value everywhere.